integration/signing_test.go

```diff
@@ -107,11 +107,11 @@ func (s *SigningSuite) TestSignVerifySmoke(c *check.C) {
 	defer os.Remove(sigOutput.Name())
 	out, err := exec.Command(skopeoBinary, "standalone-sign", "-o", sigOutput.Name(),
 		manifestPath, dockerReference, s.fingerprint).CombinedOutput()
-	c.Assert(err, check.IsNil)
+	c.Assert(err, check.IsNil, check.Commentf("%s", out))
```

Comment:
`"%s"` is actually needed; otherwise `'%'` characters in `out` would be interpreted as format string metacharacters. Also, `out` is a `[]byte`, not a `string`.